s3s.go

```diff
@@ -21,7 +21,7 @@ const (
 type FormatType int
 
 const (
-	FormatTypeJSON FormatType = iota
+	FormatTypeJSON FormatType = iota + 1
```

Comment:
If you do an en-bug like this, it will be easier to notice that it will be an invalid value instead of being JSON fallback when initialized to 0.